File: 05 SWEA/D1/Java/src/Prob2050.java

/**
* title  : 2050. 알파벳을 숫자로 변환 (D1)
* time   : 117ms
* memory : 17428kb
*/

import java.io.*;
 
public class Prob2050 {
    
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        String word = br.readLine();
        for (int i = 0; i < word.length(); i++) {
            sb.append(word.charAt(i) - 'A' + 1).append(" ");
        }

        System.out.println(sb);
    }
}

File: 05 SWEA/D1/Python/Prob2050.py

"""
title  : 2050. 알파벳을 숫자로 변환 (D1)
time   : 113ms
memory : 44288kb
"""

def solution():

    str = input().rstrip()
    for s in str:
        print(ord(s) - 64, end=" ")

solution()